Dockerfile

```diff
@@ -27,6 +27,9 @@ ENV KUBECTL /usr/local/bin/kubectl
 ENV KUBEUTIL $BUILD_HARNESS_PATH/kube-util
 WORKDIR $BUILD_HARNESS_PATH
 
+ENV USER gladly
```

Comment:
I think you also need a line like:
`RUN addgroup -S $USER && adduser -S $USER $USER`?

Reply:
Yep.... missing that feels silly....